delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeIndexingStream.scala

```diff
@@ -254,12 +303,80 @@ final class CompositeIndexingStream(
       case _: SparqlProjection        => blazeConfig
     }
 
-  private def projectionIds(view: ViewIndex[CompositeView]) =
+  private def projectionIds(view: ViewIndex[CompositeView])                          =
     CompositeViews.projectionIds(view.value, view.rev).map { case (_, _, projectionId) => projectionId }
 
+  private def streamRepeat[A](f: UIO[A], fixedRate: FiniteDuration): Stream[Task, A] =
+    Stream.eval(f) ++ Stream.repeatEval(f).metered(fixedRate)
+
+  def fetchProjectsCounts(view: CompositeView): UIO[Map[CompositeViewSource, ProjectCount]] = UIO
```

Comment:
if this method throws, what will happen to the stream? Should we attempt to recover / retry?

Reply:
The only thing that can through there is `remoteProjectsCounts`. And it uses the `HttpClient` which has its own retrying.